Packs/PasswordResetViaChatbot/Playbooks/Reset_User_Password_via_Chatbot_README.md

```diff
@@ -0,0 +1,77 @@
+| ConfirmationEmailSubject | Optional - applicable only if the "ConfirmationTarget" input is not blank.<br/>The subject of the email that will be sent to approve or disapprove the password reset. This email would be sent to the user's manager, IT staff or anyone else specified. | Action Required - User Password Reset | Optional |
+| ConfirmationEmailBody | Optional - applicable only if the "ConfirmationTarget" input is not empty.<br/>The body of the email that will be sent to approve or disapprove the password reset. This email would be sent to the user's manager, IT staff or anyone else specified. | User [PLACEHOLDER] asked to reset their password.<br/>Please allow or decline the password reset. | Optional |
+| TargetProduct | Determines where the user password will be reset. Some organizations provision the data from Okta to Active Directory, so they may choose to reset the password in Okta instead of Active Directory.<br/>Possible values are \(choose one\):<br/>- Active Directory<br/>- Okta | Okta | Optional |
+| Require2FA | Whether to require that the user authenticates using an available authentication method.<br/>Currently, this only supports the Extra Verification methods available in Okta \(Okta Verify, SMS, Security Question or Google Authenticator\). | True | Optional |
+| UserEmailSubject | The subject of the email with the password that will be sent to the user. | Password Reset | Optional |
+| UserEmailBody | The body of the email with the password that will be sent to the user. Note: the password will not be included in the email body, but in a protected ZIP file instead. | Following your request to reset your password, we've changed your password to a new temporary password.<br/>After logging in, you will be required to change the password to a new one.<br/>You can find the password in the ZIP file attached to this email. The password to the ZIP file was sent to you in a direct message on Slack or on Teams. | Optional |
+| ZipPasswordMessage | The message that will be sent to the user on Slack/Teams after their password was reset. The value of this input will be appended to the beginning of the the ZIP password that will be sent in the following manner:<br/>"The zip file password is: \[zip_password_will_be_here\]".<br/><br/>Note: there is no need to actually write the placeholder for the ZIP password. It will simply appear after your message.<br/> | Following your request to reset your password, we've sent you an email with the new password. The password is protected inside an encrypted ZIP file.<br/>The password to the ZIP file is:  | Optional |
```

Comment:
```suggestion
| ZipPasswordMessage | The message that will be sent to the user in Slack/Teams after their password was reset. The value of this input will be appended to the beginning of the the ZIP password that will be sent in the following manner:<br/>"The zip file password is: \[zip_password_will_be_here\]".<br/><br/>Note: There is no need to actually write the placeholder for the ZIP password. It will simply appear after your message.<br/> | Following your request to reset your password, we've sent you an email with the new password. The password is protected inside an encrypted ZIP file.<br/>The password to the ZIP file is:  | Optional |
```